ajax/libs/leaflet-polylinedecorator/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "leaflet-polylinedecorator",
+  "filename": "leaflet.polylineDecorator.min.js",
+  "npmName": "leaflet-polylinedecorator",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "leaflet-polylinedecorator*"
+      ]
+    }
+  ],
+  "version": "1.0.1",
+  "authors": [
+    "Benjamin",
+    "Becquet"
```

Comment:
I saw it in bower.json.
But I was still wrong.
Maybe the correct form is: ```"author": "Benjamin Becquet"```
Right?